src/sinks/aws_cloudwatch_logs/retry.rs

```diff
@@ -36,23 +36,26 @@ impl<T: Send + Sync + 'static> RetryLogic for CloudwatchRetryLogic<T> {
     fn is_retriable_error(&self, error: &Self::Error) -> bool {
```

Comment:
I suspect we don't need these specific matchers and can delegate to the shared `is_retriable_error` but didn't want to pull that into this PR.